src/main/java/uk/gov/ons/ctp/response/action/service/NotificationFileCreator.java

```diff
@@ -19,51 +19,109 @@
   private static final SimpleDateFormat FILENAME_DATE_FORMAT =
       new SimpleDateFormat("ddMMyyyy_HHmm");
 
-  private final ExportFileRepository exportFileRepository;
-
   private final Clock clock;
 
+  private ActionStateService actionStateService;
+
   private PrintFileService printFileService;
 
+  private class ExportData {
+    private List<ActionRequest> ariList;
```

Comment:
Minor, but ariList should be `arList` because action request instructions don't exist anymore

Reply:
yep good point, I copied this from elsewhere in the code but yes its badly named. I'll rename it actionRequests